doc/parameters.md

```diff
@@ -186,6 +186,29 @@ before the client is throttled by adding latency.
 When set to _true_, APIcast will log the response code of the response returned by the API backend in 3scale. In some plans this information can later be consulted from the 3scale admin portal.
 Find more information about the Response Codes feature on the [3scale support site](https://access.redhat.com/documentation/en-us/red_hat_3scale/2.saas/html/analytics/response-codes-tracking).
 
+### `APICAST_SERVICES_FILTER`
+**Value:** a regegular expression
+**Default:**: .*
+**Example:** *.example.com
```

Comment:
Will it be possible to supply multiple patterns? I totally could see that there will be two wildcards in some cases.

Reply:
Hm. Based on the code it looks like this should be a regex, but from the example, it looks like a wildcard.

Reply:
should be clear now ;-)

Reply:
`*.example.com` is not valid PCRE no?